include/net/net_pkt.h

```diff
@@ -419,12 +419,17 @@ static inline void net_pkt_set_priority(struct net_pkt *pkt,
 {
 	pkt->priority = priority;
 }
-#else
+#else /* NET_TC_COUNT == 1 */
+#define net_pkt_set_priority(...)
+
 static inline u8_t net_pkt_priority(struct net_pkt *pkt)
 {
 	return 0;
 }
-#endif
+
+#define net_pkt_set_priority(...)
```

Comment:
my mistake, I had put the macro before the inline one, then realized the order is reversed (when there are 2 inilnes), so I did reverse the order... and forget to remove the first.